### PR TITLE
[16.0][ADD] edi_sale_edifact_oca

### PR DESCRIPTION
**Pull request by QuocDuong1306**
- Not done, need to rewrite the tests
- The test of `edi_sale_ubl_oca` in v14 was failed too

### CODE REVIEW

**Comment by QuocDuong1306** [comment]
Updated test cases !

File: edi_sale_edifact_oca/demo/edi_exchange_type.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+<!-- TODO: noupdate OR better create via post init hook   -->
+    <record id="demo_edi_exc_type_edifact_order_in" model="edi.exchange.type">
+      <field name="backend_type_id" ref="edi_edifact_oca.edi_backend_type_edifact" />
```
Comment (nilshamerlinck) [inline edi_sale_edifact_oca/demo/edi_exchange_type.xml:5]:
```suggestion
      <field name="backend_id" ref="edi_edifact_oca.edi_backend_edifact_demo" />
      <field name="backend_type_id" ref="edi_edifact_oca.edi_backend_type_edifact" />
```

**Review by nilshamerlinck — CHANGES_REQUESTED** [review]
(will review again when all state related is removed)

**Comment by QuocDuong1306** [comment]
Hi @nilshamerlinck , could you have a look?

**Review by nilshamerlinck — APPROVED** [review]

**Pull request merged**